bindings/node/lib/stateMachine.js

```diff
@@ -437,13 +437,15 @@ module.exports = function (modules) {
       const dbName = databaseNamespace(ns);
       const rawCommand = bson.deserialize(command, options);
 
-      client.db(dbName).command(rawCommand, options, (err, response) => {
-        if (err) {
+      client
+        .db(dbName)
+        .command(rawCommand, options)
+        .then(response => {
```

Comment:
```typescript
client
        .db(dbName)
        .command(rawCommand, options)
        .then(
          response => {
            return callback(null, bson.serialize(response, this.options));
          },
          error => {
            callback(error, null);
          }
        );
```
We should use the two arguments of `then`, otherwise if the callback throws here we'll call it twice

Reply:
Updated.